### PR TITLE
support wasmedge on KubeEdge 

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
add operation steps for running a WasmEdge demo on KubeEdge

### CODE REVIEW

File: sig-node/wasmedge/support_wasmedge.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+WasmEdge is a lightweight, high-performance, and extensible WebAssembly runtime for cloud native, edge, and decentralized applications. It powers serverless apps, embedded functions, microservices, smart contracts, and IoT devices. WasmEdge is currently a CNCF (Cloud Native Computing Foundation) Sandbox project.
```
Comment (gy95) [inline sig-node/wasmedge/support_wasmedge.md:1]: This is not consistent with the Chinese part. we'd better keep the both the same.

File: sig-node/wasmedge/support_wasmedge_cn.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+``` 
+
+CloudCore安装完成后，使用`keadm gettoken`获取token，用于安装EdgeCore时使用。
+
+## 3. 在边缘节点安装EdgeCore
+
+由于KubeEdge v1.13.0默认使用容器运行时为containerd，因此在安装EdgeCore时可以不指定容器运行时。如果安装的是v1.13.0之前的版本，需要在执行keadm join命令时，使用`--runtimetype`和`--remote-runtime-endpoint`指定边缘节点需要的容器运行时。
+
+```
+keadm join --cloudcore-ipport="THE-EXPOSED-IP":10000 --token=$token} kubeedge-version=v1.13.0
```
Comment (gy95) [inline sig-node/wasmedge/support_wasmedge_cn.md:97]:
```suggestion
keadm join --cloudcore-ipport="THE-EXPOSED-IP":10000 --token=$token} --kubeedge-version=v1.13.0
```

File: sig-node/wasmedge/support_wasmedge.md (hunk 1 above)
Comment (Shelley-BaoYue) [reply sig-node/wasmedge/support_wasmedge.md]: > This is not consistent with the Chinese part. we'd better keep the both the same.

Ok, it has been modified

File: sig-node/wasmedge/support_wasmedge_cn.md (hunk 2 above)
Comment (Shelley-BaoYue) [reply sig-node/wasmedge/support_wasmedge_cn.md]: > 

fixed.

**Comment by kevin-wangzefeng** [comment]
/lgtm
/approve
Thanks

**Pull request merged**